zkevm-circuits/src/evm_circuit/execution/callop.rs

```diff
@@ -58,6 +59,7 @@ pub(crate) struct CallOpGadget<F> {
     is_empty_code_hash: IsEqualGadget<F>,
     one_64th_gas: ConstantDivisionGadget<F, N_BYTES_GAS>,
     capped_callee_gas_left: MinMaxGadget<F, N_BYTES_GAS>,
+    is_precompile: Cell<F>,
```

Comment:
this cell comes from nowhere. It should be constrained: is_compile == less_than(addr, 0x9) ?